File: Lecture_1/solutions/exercise3.cc
#include <iostream>

int main()
{
    int a = 1, b = 1;
    int target = 48;
    for(int n = 3; n <= target; ++n)
    {
        int fib = a + b;
        std::cout << "F("<< n << ") = " << fib << std::endl;
        a = b;
        b = fib;
    }

    return 0;
}

File: Lecture_1/solutions/exercise3.py
#!/usr/bin/env python

def main():
    a = 1
    b = 1
    target = 48
    for n in range(3, 49):
        fib = a + b
        print(f"F({n}) = {fib}")
        a = b
        b = fib


if __name__ == "__main__":
    main()
